background/ncn_boot_workflow.md

```diff
@@ -15,6 +17,27 @@ Non-compute nodes boot two ways:
 * [Reverting Changes](#reverting-changes)
 * [Locating USB Device](#locating-usb-device)
 
+<a name="determine-the-current-boot-order"></a>
+
+### Determine the current bootorder
+
+Under normal operations, the NCNs use the following boot order:
+
+1. PXE (to allow for booting with changes in BSS)
```

Comment:
Nit: make more generic, "to ensure the NCN is booting with desired images and configs."

Reply:
Fixed